jest.setup.js

```diff
@@ -1 +1,9 @@
 global.delay = (n) => new Promise((resolve) => setTimeout(resolve, n))
+
+global.tryCatch = (fn) => {
+    try {
+        fn()
+    } catch (e) {
+        //
+    }
+}
```

Comment:
Trying to assign to a proxy where the `set` trap returns false throws an exception in a node environment.